src/git.md

```diff
@@ -441,6 +441,44 @@ tradeoff. The main advantage is the generally linear commit history. This
 greatly simplifies bisecting and makes the history and commit log much easier
 to follow and understand.
 
+## Review suggestions
+
+**NOTE**: This section is for *reviewing* PRs, not authoring them.
+
+### Hiding whitespace
+
+Github has a button for disabling whitespace changes we recommend using.
```

Comment:
FWIW, I usually review with whitespace changes visible. Perhaps say "that can be useful" or something like that?